pkg/plugins/mellanox/mellanox_plugin.go

```diff
@@ -266,25 +258,6 @@ func mlnxNicFromMap(mstData map[string]string) (*mlnxNic, error) {
 	return fwData, nil
 }
 
-func parseMstconfigOutput(mstOutput string, attributes []string) (fwCurrent, fwNext map[string]string) {
-	glog.Infof("mellanox-plugin parseMstconfigOutput(): Attributes %v", attributes)
-	fwCurrent = map[string]string{}
-	fwNext = map[string]string{}
-	formatRegex := regexp.MustCompile(`(?P<Attribute>\w+)\s+(?P<Default>\S+)\s+(?P<Current>\S+)\s+(?P<Next>\S+)`)
-	mstOutputLines := strings.Split(mstOutput, "\n")
-	for _, attr := range attributes {
-		for _, line := range mstOutputLines {
-			if strings.Contains(line, attr) {
-				regexResult := formatRegex.FindStringSubmatch(line)
-				fwCurrent[attr] = regexResult[3]
-				fwNext[attr] = regexResult[4]
-				break
-			}
-		}
-	}
-	return
-}
-
```

Comment:
This is a refactor, please do that in a separate commit

Reply:
done I separate the commit